devicon.json

```
@@ -1125,6 +1125,28 @@
         "color": "#343434",
         "aliases": []
     },
+    {
+        "name": "clarity",
+        "altnames": [],
+        "tags": [
+            "programming",
+            "language"
```

```suggestion
            "language",
            "blockchain"
```

We might also consider having a tag called `smart-contracts` or something too, but that hasn't been used before, so I'm not so sure

---

Yeah, there are some technologies that `smart-contracts` tag fits perfectly on them.
Committing your changes!